### PR TITLE
add attachments to products on webshop

### PR DESCRIPTION
**Pull request by mFlayyan**

### CODE REVIEW

**Comment by hbrunn** [comment]
restarting build after corrected target branch

**Comment by hbrunn** [comment]
hmmm, the CI insists on running this with 10.0, but we need 9.0 @mFlayyan better close this PR and create a new one against 9.0 from the start, then the CI tests shouldn't fail as miserably as they do currently. (or anyone else: Do we know how to fix this?)

**Comment by pedrobaeza** [comment]
You need to amend the commit and force push to launch correctly Travis.

**Comment by mFlayyan** [comment]
I think the CI errors are unrelated since it is failing on test_website_sale_wishlist

**Review by hbrunn — APPROVED** [review]

File: website_sale_product_attachment/views/product_attachment.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (susport) [inline website_sale_product_attachment/views/product_attachment.xml:2]: change labels openerp data by odoo

**Pull request closed without merging**